### ISSUE TITLE
Unity 2020.2.0bx RenderList issues

### ISSUE THREAD

**Issue by sasa42** [alelievr/HDRP-Custom-Passes#31]
Following errors appear in Unity 2020.2.0bx 

![image](https://user-images.githubusercontent.com/1211326/124493954-84661100-ddb6-11eb-85c9-bf718e9bec0f.png)

**Comment by alelievr** [comment]
Yes, that's because the render list API was removed without being deprecated. This is being fixed on HDRP side.

**Comment by sasa42** [comment]
Thanks. Does it means it will be auto fixed in 2020.3..0b3+
I am currently in 2020.3.0b3.

**Comment by alelievr** [comment]
No, it will happen in a later version. (btw it's 2021.3, not 2020.3 :) )

**Comment by OwenProto** [comment]
is there a target HDRP version for the renderlist incompatibility?
